packages/web3/src/connect-button/connect-button.tsx

```diff
@@ -128,11 +128,11 @@ export const ConnectButton: React.FC<ConnectButtonProps> = (props) => {
     <div className={`${prefixCls}-content`}>
       <div className={`${prefixCls}-content-inner`}>
         <div className={`${prefixCls}-text`}>{buttonText}</div>
-        {avatar && (
+        {(account?.avatar || avatar) && (
           <>
             <Divider type="vertical" />
             <div className={`${prefixCls}-avatar`}>
-              <Avatar {...avatar} />
+              <Avatar src={account?.avatar} {...avatar} />
```

Comment:
```suggestion
              <Avatar {...avatar} />
```
1. 看起来似乎不需要这个src了，上面写的avatar中，如果account存在的话会配置src，不存在的情况也给了配置，从avatar中应该也都解构出来了。
2. bot提到的test可以覆盖一下
<img width="682" alt="image" src="https://github.com/ant-design/ant-design-web3/assets/117748716/34955c1e-4749-41c6-ac59-8d8a9e86f360">